config/initializers/oj.rb

```diff
@@ -0,0 +1,3 @@
+require 'oj'
```

Comment:
not sure we need this require?

Reply:
:+1: Just tested this and no we don't - [amended](https://github.com/DFE-Digital/apply-for-teacher-training/pull/5297/commits/ee53e4d61abba3ade0544b2ef77a4ba535acec41)